_clients/javascript/helpers.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+title: Helpers
+parent: JavaScript client
+nav_order: 2
+---
+
+# Helpers
+
+This section is currently incomplete. It does not yet cover all helper methods and is missing documentation of various helper options.
+{: .warning }
+
+## Bulk helper
+
+Running bulk requests can be complex due to the shape of the API, this helper aims to provide a nicer developer experience around the Bulk API.
+
+### Usage
+
+```
+const { Client } = require('@opensearch-project/opensearch')
+const documents = require('./docs.json')
+
+const client = new Client({ ... })
+
+const result = await client.helpers.bulk({
+  datasource: documents,
+  onDocument (doc) {
+    return {
+      index: { _index: 'example-index' }
+    }
+  }
+})
+
+console.log(result)
```

Comment:
Lines 35-42 represent the fields you will see in the log, correct? If so, I would separate those from the code snippet and explain them below.